test/fika/erl_translate_test.exs

```diff
@@ -143,7 +143,7 @@ defmodule Fika.ErlTranslateTest do
     ast = TestParser.expression!(str)
     result = ErlTranslate.translate_expression(ast)
 
-    assert result == {:atom, 1, :foo}
+    assert result == {:atom, 1, %Fika.Types.Atom{value: :foo}}
```

Comment:
I think this is because we use the same AST node to represent both atom values and atom types. Maybe this should be changed?
I'll look into this and maybe this will help getting rid of the struct

Reply:
Yeah, this insight led to a simpler representation :)